docker/docker-compose.yml

```diff
@@ -62,6 +65,10 @@ services:
     build:
       context: ..
       dockerfile: ./docker/bigquery-writer/Dockerfile
+    volumes:
+      - ../credentials:/code/credentials
```

Comment:
I think that makes sense, but for the interest of getting the best test out, I'd prefer not to touch this right this second.